docs/primary/topic_based_expanded_reference/GUIDE.md

```diff
@@ -1352,30 +1352,6 @@ reminder for future uses of stack:
 * stack.yaml can override the flag settings for individual packages
 * stack.yaml specifies which packages to include
 
-## Comparison to other tools
```

Comment:
I'm not certain this should be moved out of the guide. If it is:

1. @wolftune just did a significant overhaul of the text in the guide, so the updated text should be copied over.
2. We should have a link to the new file added here.